source/CMakeLists.txt

```diff
@@ -284,9 +285,19 @@ target_link_directories(appleii PUBLIC
   ${ZLIB_LIBRARY_DIRS}
   )
 
-target_compile_options(appleii PUBLIC
-  -Wno-multichar
+if (MSVC)
+  target_compile_options(appleii PUBLIC
+    /wd4566   # disable multi-character constant warning
```

Comment:
ideally we should fix this code in the upstream repo. can you propose a MR there.

Reply:
Let me see where it pukes and propose a PR